include/cuco/static_map.cuh

```diff
@@ -266,6 +287,30 @@ class static_map {
                  KeyEqual key_equal  = KeyEqual{},
                  cudaStream_t stream = 0);
 
+  /**
+   * @brief Erases keys in the range `[first, last)`.
+   *
+   * This function synchronizes `stream`.
```

Comment:
Document that this will throw if `erased` wasn't provided, (e.g., `erased/empty` are the same value)